book/src/designs/0002-parallel-verification.md

```diff
@@ -60,27 +60,10 @@ In Zebra, verification happens in the following stages:
 * **Structural Verification:** Raw block data is parsed into a block header and
   transactions. Invalid data is not representable in these structures:
   deserialization (parsing) can fail, but serialization always succeeds.
-* **Context-Free Verification:** Fields that don't depend on any context from
```

Comment:
RE: your earlier comment about wanting to break things up into smaller units of work, do you think we could move the documentation about how the chain reorgs work and transience to its own RFC that's more specific to the state service?

Reply:
I don't have a strong opinion either way.

It might be helpful to reference constraints as a concept in this RFC, but deal with the details in another RFC.

What I'd like to do is merge the bits we agree on for this RFC, and then deal with the remaining questions in future PRs. (For this RFC, or new RFCs.)

Reply:
This is now split out into #763 and #871.